client/src/components/main.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+// import components
+import Navbar from './navbar/Navbar';
+import Footer from './footer/Footer';
+import Home from './home/Home';
+import Background from './Background/Background';
+
+/**
+ * @class Main 
+ * @classdesc returns the main wrapper of the app
+ */
+class Main extends React.Component {
```

Comment:
This component will be better off a functional component.

Reply:
Oh yeah true. Thanks for the feedback.